popup/pages/AllPage.tsx

```diff
@@ -8,8 +9,32 @@ export const AllPage = () => {
   const search = useAtomValue(searchAtom);
   const entryIdToTags = useAtomValue(entryIdToTagsAtom);
 
+  const renderNoEntriesOverlay = () => {
+    return (
+      <Stack h={450} w={700} justify="center" align="center">
```

Comment:
nit: Make a `Box` that wraps around `noEntriesOverlay` in `EntryList` so the caller doesn't have to worry about sizing and remove the `h` and `w` props here.

Reply:
So you want only the text to be passed from the` page` components and the rest should be handled in the `EntryList`? Also, I should use Box instead of Stack?

Reply:
Oh no. I'm saying `renderNoEntriesOverlay` should just be a new component that takes in `title` and `subtitle` which `AllPage` and `FavoritesPage` can share. Keep using the stack, but move only the height/width props to a `Box` component in `EntryList` that wraps the render.

Reply:
Feel free to message me on Discord if it still doesn't make sense. IMO that approach is more maintainable (as it's in line with how other component libraries do things).